R/globals.R

```diff
@@ -1 +1,7 @@
-utils::globalVariables(c("rexp", "rnorm", "runif", "stats"))
+utils::globalVariables(c(
```

Comment:
These should be imported into your namespace

```
#' @importFrom stats rexp
```

which will add a line into the `NAMESPACE` file on roxygenisation